### ISSUE TITLE
idk how to use

### ISSUE THREAD

**Issue by mikeygio10** [Atomization/Events#1]
If you could i would be awesome if you made a video on how to setup or something

**Comment by Atm85** [comment]
Yes, ill make a video sometime this week, thanks for suggestion!
